spec/system/referee_interface/referee_cannot_provide_reference_if_two_already_received_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.feature 'Referee can submit reference', with_audited: true do
+  include CandidateHelper
+
+  scenario 'Referee submits a reference for a candidate with relationship, safeguarding and review page' do
+    given_the_candidate_has_requested_three_references_and_i_am_the_third_referee
+    and_the_first_referee_has_responded_with_a_reference
+    and_the_second_referee_has_responded_with_a_reference
+    then_i_receive_an_email_with_a_reference_request
```

Comment:
I think this should be `when_i_receive..`

Reply:
👍